product_pricelist_alternative/models/product_pricelist.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2024 Camptocamp (<https://www.camptocamp.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+
+
+class Pricelist(models.Model):
+    _inherit = "product.pricelist"
+
+    alternative_pricelist_ids = fields.Many2many(
+        comodel_name="product.pricelist",
+        string="Alternative pricelists",
+        relation="alternative_pricelist_rel",
```

Comment:
```suggestion
        relation="product_pricelist_alternative_rel",
```
or don't provide a value and let the orm choose the std